content/sensu-go/5.14/api/federation.md

```diff
@@ -0,0 +1,408 @@
+---
+title: "Federation API"
+description: "The federation API controls federation of Sensu clusters. Here’s a reference for the federation API in Sensu Go, including examples. Read on for the full reference."
+version: "5.14"
+product: "Sensu Go"
+menu:
+  sensu-go-5.14:
+    parent: api
+---
+
+- [The `/replicators` API endpoint](#the-replicators-api-endpoint)
```

Comment:
I've re-named this API to `etcd-replicators`, sorry about that. The current replicators we have are specific to etcd, and I suspect we might add different ones in the future, so I'd like to disambiguate them before we ship.